pkg/fission-cli/cmd/function/create_test.go

```diff
@@ -0,0 +1,40 @@
+package function
+
+import "testing"
+
+type pkgRequest struct {
+	uid    string
+	fnName string
+}
+
+func TestGeneratePackageName(t *testing.T) {
+	for _, test := range []struct {
+		name     string
+		request  *pkgRequest
+		expected int
+	}{
+		{
+			name: "name of package should be less than or equal to 63 characters, if function name is less than or equal to 26 characters",
+			request: &pkgRequest{
+				uid:    "35764439-0676-48bf-bb61-c3cdcc82b801",
```

Comment:
Instead of hardcoding we can generate UUID compatible to one we are getting in generate function.

Reply:
done